src/config.py

```diff
@@ -0,0 +1,71 @@
+import abc
+from typing import Optional
+
+
+class ConfigBase(abc.ABC):
```

Comment:
docstrings ? : What is this class for ? How is it intended to be used ?

Reply:
Added